src/engine/outfit.ts

```diff
@@ -147,12 +157,29 @@ export function avoidDaylightShavingsHelm(): boolean {
   );
 }
 
+// eslint-disable-next-line libram/verify-constants
+const candySword = $item`candy cane sword cane`;
+
+function useCandyCaneSword(): boolean {
+  if (!have(candySword)) return false;
+  if (get("instant_saveCandySword", false)) return false;
+  examine(candySword);
+  // eslint-disable-next-line libram/verify-constants
+  if (haveEffect($effect`Peppermint Rush`) >= 401) return false;
```

Comment:
What's the point of checking for the effect here if we can directly check for skill usage from the prefs?

Reply:
There isn't one; I failed to remove it. Updating.